src/commands/math/commands.js

```diff
@@ -79,6 +90,7 @@ LatexCmds.mathtt = bind(Style, '\\mathtt', 'span', 'class="mq-monospace mq-font"
 //text-decoration
 LatexCmds.underline = bind(Style, '\\underline', 'span', 'class="mq-non-leaf mq-underline"');
 LatexCmds.overline = LatexCmds.bar = bind(Style, '\\overline', 'span', 'class="mq-non-leaf mq-overline"');
+LatexCmds.dot = bind(DotRecurringDecimalStyle, '\\dot');
```

Comment:
the only reason to use `bind()` is for reusable, "abstract" classes like `VanillaSymbol` or `Style`. For classes that correspond to just one command (which is actually most non-symbols), I'd prefer inlining it all, like for example [`TextColor`](https://github.com/mathquill/mathquill/pull/561/files#diff-4a4c67b9876a02660f57ba183147e639R102). In this case that might be something like:
```js
LatexCmds.dot = P(MathCommand, function(_, super_) {
  _.init = function() {
    super_.init.call(this, '\\dot', ...
```